packages/yarnpkg-core/sources/MessageName.ts

```diff
@@ -95,7 +95,6 @@ export enum MessageName {
   NETWORK_UNSAFE_HTTP = 81,
   RESOLUTION_FAILED = 82,
   AUTOMERGE_GIT_ERROR = 83,
-  AUTOMERGE_LOCKFILE_VERSION_MISMATCH = 84,
```

Comment:
We can't remove these.

Reply:
Yep, that's why I removed it, it's a leftover from a previous implementation.